src/main/java/Duke.java

```diff
@@ -55,7 +55,7 @@ public static Boolean Exit(String args){
         }
     }
      */
-     public static boolean actionCommand(String args){
+     public static boolean shouldExit(String args){
```

Comment:
I like how you changed the name of the Boolean variable name to something that sounds natural, and an alternative could have been "isActionCommand"